packages/react-native-editor/src/index.js

```diff
@@ -117,7 +117,12 @@ const setupInitHooks = () => {
 
 let blocksRegistered = false;
 
-const setupLocale = ( locale, extraTranslations ) => {
+const setupLocale = ( {
+	locale,
+	extraTranslations,
+	acceptBlocks = [ 'core/verse', 'core/buttons' ],
```

Comment:
These would realistically default to `[]`, but I left them like this as an example. I will revise them to `[]` when addressing tests.